src/main/resources/db/migration/common/V1_213__dailylist_job_status_not_null.sql

```diff
@@ -1 +1,2 @@
+--v48
```

Comment:
Does adding this comment not invalidate the flyway schema history - checksum stuff?
Think it should be reverted

Reply:
ACCEPT.